Description: Remove the intersphinx plugin.
 Do not use the intersphinx plugin which is doing network access during
 the build.
Author: Thomas Goirand <zigo@debian.org>
Forwarded: no
Last-Update: 2014-07-28

--- a/doc/source/conf.py
+++ b/doc/source/conf.py
@@ -44,7 +44,6 @@
 extensions = ['sphinx.ext.autodoc',
               'sphinx.ext.todo',
               'sphinx.ext.coverage',
-              'sphinx.ext.intersphinx',
               # NOTE(blk-u): Uncomment the [pbr] section in setup.cfg and
               # remove this Sphinx extension when
               # https://launchpad.net/bugs/1260495 is fixed.
